app/kudos/kudos.yaml

```diff
@@ -645,7 +645,7 @@
   - memory management
   - common
   - programming
-  image: c++_scientist.svg
+  image: cpp_scientist.svg
```

Comment:
we will need to re-mint?

Reply:
Yes - I was planning on minting following the merge.